flink-libraries/flink-table/pom.xml

```diff
@@ -231,6 +231,25 @@ under the License.
 				</configuration>
 			</plugin>
 
+			<!-- build a har-with-dependencies, to be included in the 'opt' build folder -->
+			<plugin>
+				<artifactId>maven-assembly-plugin</artifactId>
+				<configuration>
+					<descriptorRefs>
+						<descriptorRef>jar-with-dependencies</descriptorRef>
```

Comment:
The jar-with-dependencies is 86MB. This would increase the distribution from 120 MB to 200 MB. 
I looked into the jar file and it contains all dependencies of `flink-table`, including the other Flink dependencies and their transitive dependencies (netty, hadoop, avro, etc.). We need to exclude all of that and only keep those dependencies that we really need.